Modules/Core/Common/include/itkBooleanStdVector.h

```diff
@@ -0,0 +1,43 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkBooleanStdVector_h
+#define itkBooleanStdVector_h
+
+#include "itkIntTypes.h"
+#include <vector>
+
+namespace itk
+{
+
+/** The element type of `BooleanStdVectorType`. Should be treated like a Boolean type. */
+using BooleanType = uint8_t;
```

Comment:
Why not put this in https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkIntTypes.h?

Reply:
> Why not put this in https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkIntTypes.h?

Well, I would like to avoid that ITK has its own Boolean type for general use. I would prefer to "in general" keep using the built-in `bool` type.  This `BooleanType` typedef is _only_ intended to define the element type of `BooleanStdVectorType`.

We could also drop this `BooleanType` typedef entirely, and only define `BooleanStdVectorType = std::vector<uint8_t>`. Would that be fine to you as well?

Reply:
> drop this BooleanType typedef entirely

That makes more sense if it is only going to be used for `std::vector`.